specs/experimental/contracts/L2/l2-cross-domain-messenger.md

```diff
@@ -52,16 +50,6 @@ A message sent from one domain (L1 or L2) to another, containing a target addres
 optional ETH value. Messages are encoded with metadata including sender address and nonce for authentication and
 replay protection.
 
-### Message Hash
```

Comment:
devinai You need to leave this definition in one place, either here or on the L1 side

Reply:
devinai Message Hash as used in CrossDomainMessenger is not the same as the one in L2ToL1CrossDomainMessenger